setup.py

```diff
@@ -47,7 +47,7 @@ def find_version(*file_paths):
     'tokenizers==0.8.1',
     'click>=7.0',  # Dependency of youtokentome
     'youtokentome>=1.0.6',
-    'fasttext>=0.9.2'
+    'fasttext==0.9.1'  # Fix to 0.9.1 due to https://github.com/facebookresearch/fastText/issues/1052
```

Comment:
can we do `!=` to blacklist some versions instead? my hope is that our CI can catch installation problems as soon as they happen. if you adopt this approach, then we should also specify any minimum version requirement if we have any.

Reply:
Should we do `fasttext>=0.9.1, !=0.9.2` ? Also, the problem is actually related to the fact that the latest fasttext does not work on SageMaker.